### PR TITLE
Pass local x to staggered assemblers

### PR DESCRIPTION
**Pull request by endJunction**
Again a small extraction from the #2688

### CODE REVIEW

File: ProcessLib/PhaseField/PhaseFieldFEM-impl.h [hunk 1]
```diff
@@ -22,52 +22,46 @@ template <typename ShapeFunction, typename IntegrationMethod,
...
+        const double dx_dx, int const process_id,
         std::vector<double>& local_M_data, std::vector<double>& local_K_data,
         std::vector<double>& local_b_data, std::vector<double>& local_Jac_data,
-        LocalCoupledSolutions const& local_coupled_solutions)
+        LocalCoupledSolutions const& /*local_coupled_solutions*/)
 {
     // For the equations with phase field.
     if (process_id == 1)
     {
         assembleWithJacobianPhaseFiledEquations(
```
Comment (TomFischer) [inline ProcessLib/PhaseField/PhaseFieldFEM-impl.h]: I know that this is not touched by the PR, but shouldn't that be
```suggestion
        assembleWithJacobianPhaseFieldEquations(
```
?

Comment (endJunction) [reply ProcessLib/PhaseField/PhaseFieldFEM-impl.h]: :eagle: :eye: :trophy:

Comment (TomFischer) [reply ProcessLib/PhaseField/PhaseFieldFEM-impl.h]: I didn't even see the formatting is broken. My point was the name of the function (Filed vs. Field).

Comment (endJunction) [reply ProcessLib/PhaseField/PhaseFieldFEM-impl.h]: Ah, OK. I'll push the change directly to master. :+1:

**Pull request merged**